### PR TITLE
fullscreen form view should always be in edit mode

### PR DESCRIPTION
**Pull request by msh-odoo**
Task:https://www.odoo.com/web#id=2146856&action=333&active_id=1251&model=project.task&view_type=form&menu_id=4720

Pad: https://pad.odoo.com/p/r.938a0bb2310a8f21fa9584a4bd15a9a3

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/js/views/form/form_controller.js [hunk 1]
```diff
@@ -50,15 +51,16 @@ var FormController = BasicController.extend({
      * This hook is called when a form view is restored (by clicking on the
      * breadcrumbs). In general, we force mode back to readonly, because
      * whenever we leave a form view by stacking another action on the top of
-     * it, it is saved, and should no longer be in edit mode. However, there is
-     * a special case for new records for which we still want to be in 'edit'
+     * it, it is saved, and should no longer be in edit mode. However, there are
+     * few special case, new records for which we still want to be in 'edit'
```
Comment (MiquelRForgeFlow) [inline addons/web/static/src/js/views/form/form_controller.js]:
```suggestion
     * few special cases: new records for which we still want to be in 'edit'
```

File: addons/web/static/src/js/views/form/form_controller.js [hunk 2]
```diff
@@ -50,15 +51,16 @@ var FormController = BasicController.extend({
...
      * as no record has been created (changes have been discarded before
-     * leaving).
+     * leaving), fullscreen mode for which we want to be in mode which is
```
Comment (MiquelRForgeFlow) [inline addons/web/static/src/js/views/form/form_controller.js]:
```suggestion
     * leaving) and fullscreen mode for which we want to be in mode which is
```

**Comment by lathama** [comment]
@msh-odoo knowing that 13 is no longer updated, is this PR still needed?

**Comment by C3POdoo** [comment]
Dear @msh-odoo,

Thank you for your contribution but the version 13.0 is no longer supported.
We only support the last 3 stable versions so no longer accepts patches into this branch.

We apology if we could not look at your request in time.
If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

_This is an automated message._

**Pull request closed without merging**